src/commands/lambda/function.ts

```diff
@@ -40,6 +40,46 @@ export interface InstrumentationSettings {
   tracingEnabled: boolean
 }
 
+const MAX_LAMBDA_STATE_CHECKS = 3
+
+/**
+ * Waits for n ms
+ * @param ms
+ * @returns
+ */
+const wait = (ms: number): Promise<void> => new Promise((res) => setTimeout(res, ms))
+
+/**
+ *
+ * @param config
+ * @param functionArn
+ * @param attempts
+ * @returns bool
+ */
+const checkLambdaState = async (
```

Comment:
Maybe rename to `isLambdaActive` or `isUpdatable`. `checkLambdaState` isn't a clear name, especially given that it returns a boolean.